### DISCUSSION TITLE
GPT Meet

### DISCUSSION THREAD

**Discussion by MuiseDestiny** [MuiseDestiny/zotero-gpt#96] [Ideas]
Hi, 我是 Zotero GPT 开发者。

至今，插件主体功能基本完成，已成功 Meet Zotero。

目前插件正在正在 Meet Zotero Better Notes。

未来会 Meet Zotero Style, Zotero Reference, 茉莉花 等等。

以前可能很难的事情，比如名和姓的拆分，期刊缩写转为全称，参考文献标题作者期刊解析，做笔记时候的归纳总结等等，有了GPT都会变的轻而易举。

插件的初衷是辅助你对 Zotero 的使用，让你在使用时充满惊喜，如果能提升效率那就更好了。

大家有任何想结合的插件可以在此评论，虽然不一定第一时间做，但如果你的方案与我们初衷契合我们会很快做出来。

**Comment by Bibliocr** [comment]
how do i install inside zotero?

**Comment by LynChenYe** [comment]
请问如何将回答的语言设置成英文？或者像gpt一样按照指令以特定语言回答？ 谢谢
